WalletWasabi.Tests/LiveServerTests.cs

```diff
@@ -0,0 +1,161 @@
+using Xunit;
+
+namespace WalletWasabi.Tests
+{
+	[Collection("LiveServerTests collection")]
+	public class LiveServerTests : IClassFixture<SharedFixture>
+	{
+		private SharedFixture SharedFixture { get; }
+
+		private LiveServerTestsFixture LiveServerTestsFixture { get; }
+
+		public LiveServerTests(SharedFixture sharedFixture, LiveServerTestsFixture liveServerTestsFixture)
+		{
+			SharedFixture = sharedFixture;
+			LiveServerTestsFixture = liveServerTestsFixture;
+		}
+
+		#region Blockchain
+
+		[Theory]
+		[InlineData(NetworkType.Mainnet)]
+		[InlineData(NetworkType.Testnet)]
+		public async Task GetFeesAsync(NetworkType networkType)
+		{
+			using (var client = new WasabiClient(LiveServerTestsFixture.UriMappings[networkType]))
+			{
+				var feeEstimationPairs = await client.GetFeesAsync(1000);
+
+				Assert.True(feeEstimationPairs.NotNullAndNotEmpty());
+			}
+		}
+
+		[Theory]
+		[InlineData(NetworkType.Mainnet)]
+		[InlineData(NetworkType.Testnet)]
+		public async Task GetFiltersAsync(NetworkType networkType)
+		{
+			using (var client = new WasabiClient(LiveServerTestsFixture.UriMappings[networkType]))
+			{
+				var filterModel = IndexDownloader.GetStartingFilter(Network.GetNetwork(networkType.ToString()));
+
+				var filters = await client.GetFiltersAsync(filterModel.BlockHash, 2);
+
+				Assert.True(filters.NotNullAndNotEmpty());
+				Assert.True(filters.Count() == 2);
+			}
+		}
+
+		[Theory]
+		[InlineData(NetworkType.Mainnet)]
+		[InlineData(NetworkType.Testnet)]
+		public async Task GetAllRoundStatesAsync(NetworkType networkType)
+		{
+			using (var client = new SatoshiClient(LiveServerTestsFixture.UriMappings[networkType]))
+			{
+				var states = await client.GetAllRoundStatesAsync();
+				var noRegisteredPeers = states.All(s => s.RegisteredPeerCount == 0);
+
+				Assert.True(noRegisteredPeers);
+				Assert.True(states.NotNullAndNotEmpty());
+				Assert.True(states.Count() == 2);
+			}
+		}
+
+		#endregion Blockchain
+
+		#region ChaumianCoinJoin
+
+		[Theory]
+		[InlineData(NetworkType.Mainnet)]
+		[InlineData(NetworkType.Testnet)]
+		public async Task RegisterAliceInputThenUnConfirmAsync(NetworkType networkType)
+		{
+			(BitcoinPubKeyAddress activeOutputAddress, BitcoinPubKeyAddress changeOutputAddress, string blindedDataHex, string proof, List<TxoRef> utxos) = LiveServerTestsFixture.GetAliceInputData(networkType);
+
+			// blinded activeOutputAddress.ScriptPubKey
+			byte[] blinded = ByteHelpers.FromHex(blindedDataHex);
+
+			var inputProofModels = utxos.Select(txrf => new InputProofModel
+			{
+				Input = txrf,
+				Proof = proof // blinded.BlindedData signed with private key owning utxos
+			});
+
+			var aliceClient = await AliceClient.CreateNewAsync(changeOutputAddress, blinded, inputProofModels, LiveServerTestsFixture.UriMappings[networkType]);
+
+			Assert.NotNull(aliceClient?.RoundId);
+			Assert.NotNull(aliceClient?.UniqueId);
+			Assert.NotNull(aliceClient?.BlindedOutputSignature);
```

Comment:
`aliceClient` can never be `null`. In fact, two lines below you assume (correctly) that `aliceClient` is not null.  So, remove the null coalescing operator.

Reply:
Done.